src/megaclient.cpp

```diff
@@ -1223,9 +1236,31 @@ void MegaClient::exec()
                             btcs.reset();
                             break;
                         }
+                        else
+                        {
+                            if (fetchingnodes)
+                            {
+                                fnstats.eagains++;
+                            }
+                        }
 
                     // fall through
                     case REQ_FAILURE:
+                        if (fetchingnodes)
```

Comment:
No need for 2 blocks, just `if (fetchingnodes && pendingcs->httpstatus != 200)` :)